playbooks/byo/openshift-etcd/config.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Create initial host groups for localhost
```

Comment:
The first commit actually belongs to https://github.com/openshift/openshift-ansible/pull/3817. Once that get merged, the config.yml will be the way as you suggest.